commit.go

```diff
@@ -145,7 +151,9 @@ func (c *Commit) RawData() []byte {
 		fmt.Fprintln(buf, line)
 	}
 	fmt.Fprintf(buf, "\n%s", c.Message)
-	return buf.Bytes()
+	c.rawdata = buf.Bytes()
```

Comment:
Needs a lock over this whole function to be threadsafe

Reply:
Ok, I used a [Once](https://golang.org/pkg/sync/#Once) to fix this.